src/components/common/quantityInput/QuantityInput.tsx

```diff
@@ -60,9 +58,19 @@ const QuantityInput = ({
     }
   }, [ucum, ucumUnits]);
 
+  const valueFn = (value: any) => {
+    if (value?.code) {
+      return value.code;
+    } else {
```

Comment:
we can remove else here and can directly be written as 

 if (value?.code) {
      return value.code;
    } 
    return value